server/sonar-server/src/main/java/org/sonar/server/platform/ws/ChangeLogLevelAction.java

```diff
@@ -48,6 +48,7 @@ public void define(WebService.NewController controller) {
       .setDescription("Temporarily changes level of logs. New level is not persistent and is lost " +
         "when restarting server. Requires system administration permission.")
       .setSince("5.2")
+      .setPost(true)
```

Comment:
As UI works in 5.2, I suppose that it sends a GET instead of a POST ? In this case UI must be fixed with this commit. And a JIRA ticket to be created. Correct ?